### ISSUE TITLE
Chracter completely stuck as a ghost with no way to res. GM ticket open for over half hour.

### ISSUE THREAD

**Issue by Moosestuf** [dalaranwow/dalaran-wow#4634]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Character cannot get out of a particular ditch @ 60.23 73.32

**Expected behaviour**: Tell us what should happen instead
There should never be a spot where a ghost form cannot get out of, it defeats the purpose of having to run to you body if it can lock you into a neverending prison. 

**Steps to reproduce the problem**:

1. Walk.
2. Get stuck.
3. Profit.

**Include proofs for this behaviour**
I am currently stuck here, that should be proof enough for a fix, and a high priority one at that as I am a premium member and have paid money to play this game on this server. 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
The issue tracker is only for server-side and game mechanics problems. Write an in-game ticket and we will help you as soon as possible. 

It is not a bug, ghosts aren't supposed to fly if you walk into a spot where you get stuck. That's why an unstuck feature exists. 

Closing up
